src/pages/resultsView/survival/SurvivalUtil.spec.tsx

```diff
@@ -312,8 +3142,8 @@ describe('SurvivalUtil', () => {
 
         it('returns correct stats for the example data', () => {
             assert.deepEqual(
-                getStats(examplePatientSurvivals, exampleEstimates),
-                [6, 3, '182.19']
+                getStats(examplePatientSurvivals, exampleSurvivalSummaries),
+                [6, 3, '182.19 (63.83 - NA)']
```

Comment:
so you computed the confidence intervals using R and you're checking your code against it?

Reply:
Thanks for mention this, I do checked the new dataset I put in again R package. But I checked our previous test again and found out that our previous dataset was not correct since we didn't sort the `survival data` before passing them into the function, thus we got incorrect estimates and median, I am working on fixing the tests.

Reply:
tests fixed, also compared with survival package, get the same result.

```
Call: survfit(formula = Surv(time, status) ~ 1)

      n  events  median 0.95LCL 0.95UCL 
    6.0     3.0    63.8    63.8      NA 
```

```
Call: survfit(formula = Surv(time, status) ~ 1)

   time n.risk n.event survival std.err lower 95% CI upper 95% CI
   0.09      5       1      0.8   0.179       0.5161            1
  63.83      2       1      0.4   0.297       0.0935            1
 182.19      1       1      0.0     NaN           NA           NA
```